cmake/modules/FindGLEW.cmake

```diff
@@ -62,6 +62,12 @@ if (WIN32)
         PATH_SUFFIXES
             Release/${ARCH}
         DOC "The GLEW library")
+
+    if ("${GLEW_LIBRARY}" MATCHES "glew32s(d|)")
+        add_definitions("/DGLEW_STATIC")
```

Comment:
Yeah, seems wrong -- and thinking about this more this morning, I believe the real problem is static linkage of Glew into a dynamic shared library. We should really just be linking Glew dynamically, then everything would work as expected... aside from the linker errors I get from dynamic linking when combining USD and OpenSubdiv (but that's just something that needs to be fixed).